x-pack/plugins/uptime/public/components/monitor/__snapshots__/monitor_charts.test.tsx.snap

```diff
@@ -1,93 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
-
```

Comment:
Are we adding alternative test for removing this snapshot?

Reply:
This was removed accidentally. Thanks for the catch.